forge/test/mlir/test_ops.py

```diff
@@ -749,7 +749,7 @@ def forward(self, x):
 
     co_out = [co.to("cpu") for co in co_out]
     fw_out = [fw_out] if isinstance(fw_out, torch.Tensor) else fw_out
-    assert all([compare_with_golden_pcc(golden=fo, calculated=co, pcc=0.99) for fo, co in zip(fw_out, co_out)])
+    assert all([compare_with_golden(golden=fo, calculated=co, pcc=0.99) for fo, co in zip(fw_out, co_out)])
```

Comment:
This change is unnecessary...

Reply:
Updated. Thanks for the comment @pilkicTT